### ISSUE TITLE
Windows WSL - command 'extension.Db2Connect' not found

### ISSUE THREAD

**Issue by belfortmaycon** [ibmdb/vscode-extension#76]
- Platform: Windows WSL 2 - Ubuntu 24.04
- vscode Version: 1.94.2 (user setup)
- Is node.js and npm installed: yes
- node.js and npm Version: Node.js: 20.16.0
- Take a screen shot of VScode(Help->About) information and paste it:
Version: 1.94.2 (user setup)
Commit: 384ff7382de624fb94dbaf6da11977bba1ecd427
Date: 2024-10-09T16:08:44.566Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631

Steps to Reproduce:

1. Install Db2 Connect from Extension Marketplace
2. Open the DB2 View
3. Click on + to add connection
4. Error message: command 'extension.Db2Connect' not found

**Comment by bimalkjha** [comment]
@belfortmaycon Please use option `View ->Command Pallette -> Db2: Rebuild Native Module`. If it fails, restart the vscode and try using the extension. It will try to rebuild the native module now again. Open "View -> Output" and select `Db2Connect` under `Task`. Copy all output and paste here. Thanks.

**Comment by belfortmaycon** [comment]
Hi @bimalkjha
I tried the suggested steps but still got the same error. There is no Db2Connect in the Output task list.

I tried it on Windows, and it works fine, but I need it on WSL.

**Comment by bimalkjha** [comment]
@belfortmaycon It seems some OS component is missing in your system. I would suggest to run below commands on terminal of your ubuntu and share complete output here:
```
cd $HOME/.vscode/extensions/ibm.db2connect-2.3.1/node_modules/ibm_db    # correct path as per your system
node -v
make -v
gcc --version
npm install
```
Thanks.

**Comment by bimalkjha** [comment]
@belfortmaycon I tried to reproduce the issue and can see below error when tried to use this extension:

![image](https://github.com/user-attachments/assets/9421c6c3-dcfc-4310-bd12-e6f86410340c)

After checking `Output` for `Extension Host(Remote)` in VSCode, we can see an error:
```
2024-10-26 13:52:05.330 [info] ExtensionService#_doActivateExtension IBM.db2connect, startup: false, activationEvent: 'onView:db2connect'
2024-10-26 13:52:05.393 [error] Activating extension IBM.db2connect failed due to an error:
2024-10-26 13:52:05.393 [error] TypeError: Cannot read properties of undefined (reading 'split')
	at Object.<anonymous> (/home/bimal/.vscode-server/extensions/ibm.db2connect-2.3.1/extension.js:18:51)
```
db2connect extension in a electron application. To know the version of electron, db2connect extension code tries to read value of `process.versions.electron`, but there is no electron under process.versions and hence error.

After google, I found below info:
![image](https://github.com/user-attachments/assets/94a80cf6-3f2a-4cd9-9657-57c1c847cc75)

I think info on below two links may help you:
https://www.beekeeperstudio.io/blog/building-electron-windows-ubuntu-wsl2
https://gist.github.com/caseywatts/9700b402b6b51d1d6af9f0b206739770
Thanks.
